### ISSUE TITLE
Wrong implementation of attached properties

### ISSUE THREAD

**Issue by tom-englert** [icsharpcode/AvalonEdit#270]
e.g. `FoldingMargin.SelectedFoldingMarkerBackgroundBrushProperty` and all others in `FoldingMargin`, but maybe more?

actual:
```c#
    public static readonly DependencyProperty SelectedFoldingMarkerBackgroundBrushProperty = DependencyProperty.RegisterAttached(nameof (SelectedFoldingMarkerBackgroundBrush), typeof (Brush), typeof (FoldingMargin), (PropertyMetadata) new FrameworkPropertyMetadata((object) Brushes.White, FrameworkPropertyMetadataOptions.Inherits, new PropertyChangedCallback(FoldingMargin.OnUpdateBrushes)));

    /// <summary>
    /// Gets/sets the Brush used for displaying the background of selected folding markers.
    /// </summary>
    public Brush SelectedFoldingMarkerBackgroundBrush
    {
      get => (Brush) this.GetValue(FoldingMargin.SelectedFoldingMarkerBackgroundBrushProperty);
      set => this.SetValue(FoldingMargin.SelectedFoldingMarkerBackgroundBrushProperty, (object) value);
    }
```
but correct would be:
```c#
public static readonly DependencyProperty SelectedFoldingMarkerBackgroundBrushProperty = DependencyProperty.RegisterAttached(nameof (SelectedFoldingMarkerBackgroundBrush), typeof (Brush), typeof (FoldingMargin), (PropertyMetadata) new FrameworkPropertyMetadata((object) Brushes.White, FrameworkPropertyMetadataOptions.Inherits, new PropertyChangedCallback(FoldingMargin.OnUpdateBrushes)));

public static void SetSelectedFoldingMarkerBackgroundBrush(DependencyObject element, Brush value)
{
	element.SetValue(SelectedFoldingMarkerBackgroundBrushProperty, value);
}

public static Brush GetSelectedFoldingMarkerBackgroundBrush(DependencyObject element)
{
	return (Brush)element.GetValue(SelectedFoldingMarkerBackgroundBrushProperty);
}

```
Without the two static methods attached properties are not usable in XAML, but only in code:

![image](https://user-images.githubusercontent.com/10229954/111177416-7c395a00-85aa-11eb-8a4e-d97a50bb06e5.png)

**Comment by siegfriedpammer** [comment]
@tom-englert can you double check to make sure I didn't make a mistake? Thank you very much!
Then @christophwille please create an new release for AvalonEdit, thanks!

**Comment by christophwille** [comment]
Released as 6.1.1
